bin/llvm-kompile-clang

```diff
@@ -195,7 +195,7 @@ else
 fi
 
 if $link; then
-  run @CMAKE_CXX_COMPILER@ -Wno-override-module -Wno-return-type-c-linkage "$modopt" "${files[@]}" \
+  run @CMAKE_CXX_COMPILER@ -std=c++17 -Wno-override-module -Wno-return-type-c-linkage "$modopt" "${files[@]}" \
```

Comment:
https://github.com/runtimeverification/llvm-backend/pull/696 subsumes this change

Reply:
Yes, I will revert after it is merged.

Reply:
It turned out that this flag is needed even after merging master (including PR [#696](https://github.com/runtimeverification/llvm-backend/pull/696). Removing this caused multiple (62) integration tests to fail, so I kept it.

Reply:
Addressed.